.github/workflows/release-notes.yml

```diff
@@ -60,18 +61,20 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           owner: 'corgibytes'
           repository: 'freshli-lib'
-          milestone: 'v${{ steps.gitversion.outputs.majorMinorPatch }}' 
+          milestone: 'v${{ steps.gitversion.outputs.majorMinorPatch }}'
 
       # This will generate the change log for all the GitHub Releases, feature
       # is not included in the GitReleaseManager action yet.
       - name: Generate Change Log
         run: |
-          dotnet-gitreleasemanager export --token ${{ secrets.GITHUB_TOKEN }} -o 'corgibytes' -r 'freshli-lib' -f 'CHANGELOG.md'
+          dotnet-gitreleasemanager export --token ${{ secrets.PERSONAL_ACCESS_TOKEN }} -o 'corgibytes' -r 'freshli-lib' -f 'CHANGELOG.md'
           cat CHANGELOG.md
 
       - uses: stefanzweifel/git-auto-commit-action@v4
         name: Commit Change Log and if it Changed
         with:
           commit_message: Committing auto generated change log.
           file_pattern: CHANGELOG.md
+          commit_user_name: M. Scott Ford (as bot)
+          commit_user_email: scott@corgibytes.com
```

Comment:
Just curious what happens if the email does not match the PAT token account?

Reply:
According to the discussions that I read, it has to match in order for it to actually work.